src/pages/CovidStatDetails.js

```diff
@@ -16,69 +16,69 @@ const CovidStatDetails = () => {
         <div className="text-left">
           <FontAwesomeIcon icon={faArrowAltCircleLeft} size="2x" className="text-primary mb-2" onClick={() => navigate(-1)} style={{ cursor: 'pointer' }} />
         </div>
-        <Card className="text-center bg-light p-3" style={{ width: '40rem' }}>
-          <Card.Img src={detailArr.flag} variant="top" style={{ width: '100%' }} />
+        <Card className="text-center bg-light" style={{ width: '40rem' }}>
+          <Card.Img src={detailArr.flag} className="p-3" variant="top" style={{ width: '100%' }} />
           <Card.Body>
             <Card.Title><h1 className="text-primary">{detailArr.country}</h1></Card.Title>
```

Comment:
### Your Title
![image](https://user-images.githubusercontent.com/93039359/227446253-3379ada7-692f-4878-afb3-db34088653d2.png)

### Design Title
![image](https://user-images.githubusercontent.com/93039359/227446279-07036728-35cf-4cbf-b105-43b6926d9795.png)

- [ ] As suggested by the previous reviewer that you should match the typography of your title(country name) 👍 so it would be great if you make your title more bold to match the typography of the design to make your project more attractive and professional 😊